addOns/authhelper/src/main/javahelp/org/zaproxy/addon/authhelper/resources/help/contents/auth-report-json.html

```diff
@@ -121,6 +121,40 @@ <H3>Summary</H3>
 		</tr>
 	</table>
 
+	<H4>Failure Details</H3>
+	The following failure detail items are used:
+
+	<table>
+		<tr>
+			<th align="left">Key</th>
+			<th align="left">Description</th>
+		</tr>
+		<tr>
+			<td>overall</td>
+			<td>All authentication elements passed yet success was deemed a failure in the end. Are you sure the credentials are correct? Are you certain the verification configuration is correct?</td>
```

Comment:
I like the idea of suggesting things to look at, but these look too specific to me. 
I dont think we should be using this `overall` failure in any case, so we wont know how we got here.

Reply:
Then Auth Helper needs to be reworked to prevent this situation from happening..... which is outside the scope of this PR.
As things are, we can fail overall, while passing all four elements.

Reply:
I've made the text less specific.

Reply:
I think it's better for us to acknowledge this than for people to think the report is broken. Or to mislead them into thinking auth is working.